recipes/cgsi-gsoap/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cgsi-gsoap" %}
+{% set version = "1.3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.cern.ch/dmc/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: 5fe3d8741f906dc77a13e61871d18bcf0a6deab6dd70fb0b0c977b4ef61ff924
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    # Only pin on Linux as static libaries are used on macOS
+    - {{ pin_subpackage(name, max_pin='x.x') }}  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+  host:
+    - gsoap =2.7
+    - voms
+    - gct
+    - openssl 1.0.2r
```

Comment:
I'm using openssl 1.0 as I'm building something which needs it so I'll migrate over to 1.1 in the feedstock.

I've checked and `run_export` is used correctly in `openssl=1.0.2r`.